crates/aptos-inspection-service/src/server/metrics.rs

```diff
@@ -6,9 +6,47 @@ use crate::server::{
     utils,
     utils::{CONTENT_TYPE_JSON, CONTENT_TYPE_TEXT},
 };
+use aptos_config::config::NodeConfig;
 use hyper::{Body, StatusCode};
 use prometheus::TextEncoder;
 
+// The metric key for the consensus execution gauge
+const CONSENSUS_EXECUTION_GAUGE: &str = "aptos_state_sync_consensus_executing_gauge{}";
+
+/// Handles a consensus health check request. This method returns
+/// 200 iff the node is currently participating in consensus.
```

Comment:
```suggestion
/// 200 if the node is currently participating in consensus.
```

Reply:
Aah, this was meant to be "if and only if" 😄 https://www.merriam-webster.com/dictionary/iff

Reply:
> Aah, this was meant to be "if and only if" 😄 [merriam-webster.com/dictionary/iff](https://www.merriam-webster.com/dictionary/iff)

I recommend simply using if and only if in that case. I doubt that even half of our eng team knows the `iff` thingy.

Reply:
SGTM. Changed it to `if`. Don't feel as strongly as you do 😄